pyvlx/pyvlx.py

```diff
@@ -76,7 +79,10 @@ async def send_frame(self, frame: FrameBase) -> None:
     async def disconnect(self) -> None:
         """Disconnect from KLF 200."""
         # If the connection will be closed while house status monitor is enabled, a reconnection will fail on SSL handshake.
-        await house_status_monitor_disable(pyvlx=self)
+        try:
+            await self.klf200.house_status_monitor_disable(pyvlx=self, timeout=1)
+        except Exception:
```

Comment:
Cant we catch a more specific Exception here?

Reply:
I would start with (OSError, PyVLXException)